src/odemis/driver/symphotime.py

```diff
@@ -574,20 +574,20 @@ def __init__(self, name, role, children, host, port=DEFAULT_PORT, **kwargs):
                                 "Server is connected to the network, turned "
                                 "on, and correctly configured." % (host, port))
 
+
```

Comment:
is the extra line here necessary? I think one empty line after rising an exception is enough?